frontend/main-app/src/components/NotFound/NotFound.tsx

```diff
@@ -0,0 +1,21 @@
+import classes from "./NotFound.module.css";
+import { Heading, Link, Paragraph } from "@digdir/design-system-react";
+import React from "react";
+import notFoundImage from "./notFound.png";
+export const NotFound: React.FC = () => {
+  return (
+    <main className={classes.card}>
+      <div className={classes.text}>
+        <Heading spacing>Page not found</Heading>
+        <Paragraph>We could not find the page you were looking for.</Paragraph>
```

Comment:
Could you perhaps make this sentence more context driven?
For example; 
if an user try to navigate to a form with an invalid id, instead of "Page not found" and "We could not find the page..." the "page" could be replaced with "form".